docs/userguide/networking/get-started-overlay.md

```diff
@@ -24,7 +24,7 @@ some pre-existing conditions before you can create one. These conditions are:
 Though Docker Machine and Docker Swarm are not mandatory to experience Docker
 multi-host networking, this example uses them to illustrate how they are
 integrated. You'll use Machine to create both the key-value store
-server and the host cluster. This example creates a Swarm cluster.
+server and the host cluster. This example creates a swarm.
```

Comment:
not sure if we want to change all the "Docker Swarm" docs. This doesn't refer to "swarm mode"